Procfile

```diff
@@ -36,10 +36,9 @@ celery_scheduler: REMAP_SIGTERM=SIGQUIT newrelic-admin run-program celery beat -
 # to the `store_pulse_{pushes,jobs}` queues for `worker_store_pulse_data` to process.
 # NB: These should not be scaled up to more than 1 of each.
 # TODO: Merge these two listeners into one since they use so little CPU each (bug 1530965).
-pulse_listener_pushes: newrelic-admin run-program ./manage.py pulse_listener_pushes
-pulse_listener_tasks: newrelic-admin run-program ./manage.py pulse_listener_tasks
+pulse_listener_tasks_pushes: newrelic-admin run-program ./manage.py pulse_listener
```

Comment:
I'm thinking that this change is scary enough that we should keep all 3 ingestions (the old two and the new one).
In production we will *not* run the new Pulse ingestion and we can run it on the staging app the new way for a bit.
We will need to use a `misc/compare_pushes.py` to make sure that both systems end up with the same data.

Reply:
You will need to undo deleting the original `pulse_listener_*.py` scripts.

Reply:
Ohh. Okay. Will do so!